File: 0058-length-of-last-word/solution.java
class Solution {
    public int lengthOfLastWord(String s) {
        s= s.trim();
        int i=s.length()-1;
        int wordLen=0;
        while(i>=0){
            if(s.charAt(i) == ' ')break;
            else wordLen++;

            i--;
        }
        return wordLen;
    }
}

File: 0058-length-of-last-word/solution.cpp
class Solution {
public:
    int lengthOfLastWord(string s) {
        int e = -1;

        for(int i=s.length()-1; i>=0; i--){
            if(e == -1 && s[i] != ' ') e = i;
            else if(s[i] == ' ' && e != -1) return e-i;
        }

        return e-0+1;
    }
};
